industrial_taxonomy/pipeline/glass_clusters/flow.py

```diff
@@ -1,24 +1,27 @@
 """Flow to cluster glass companies into text clusters
 """
+import logging
 
 from typing import Dict, List, Tuple
-from metaflow import FlowSpec, JSONType, project, step, Parameter
+from metaflow import FlowSpec, JSONType, current, project, step, Parameter
 from industrial_taxonomy.pipeline.glass_clusters.hSBM_Topicmodel.sbmtm import sbmtm
```

Comment:
```suggestion
from sbmtm import sbmtm
```

- This is using the out-dated cloned repo
- `industrial_taxonomy.pipeline.glass_clusters` isn't a module, there's no `__init__.py`

Reply:
This one isn't resolved